pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/pex/Expression.java

```diff
@@ -18,6 +18,6 @@ public class Expression {
     String text;
 
     public Expression(final String text) {
```

Comment:
I assume this constructor might be used for test purposes only. If yes, I suggest annotating the class with the `SuperBuilder` annotation and using the builder pattern in tests. I did it for UserDto and StudyDto partially. It helped in cases when a new column was added (or an existing one was removed) to the database's table. Once you update DTO correspondingly, AllArgsConstructor will have a different set of arguments which means that you'll have to update all the tests that are using this constructor. Using the builder pattern in the test on the other hand will not require you to review all of them and fix compilation errors.

Reply:
You are correct, that constructor is only used in tests, but it is also the only required field for that type. As for the use of a Builder pattern here- I disagree. Having to review all of the uses and fix compilation errors is a positive, not a negative. The lombok builders are prone to runtime errors and are incapable of identifying required fields at compile time without heavy customization of the codegen. Having to fix compilation errors when something changes is a good thing.